<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerrainWindow</class>
 <widget class="QWidget" name="TerrainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="subNavigationFrame">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QCommandLinkButton" name="showFormationTab">
        <property name="text">
         <string>Formation</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/toplevel-navigation/resources/sketchy_cc-attribution-noncommercial_mountain-sketch.png</normaloff>:/toplevel-navigation/resources/sketchy_cc-attribution-noncommercial_mountain-sketch.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="description">
         <string>Shape of Terrain.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="showTexturingTab">
        <property name="text">
         <string>Colouring</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/toplevel-navigation/resources/oxygen_gpl_texturing.png</normaloff>:/toplevel-navigation/resources/oxygen_gpl_texturing.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="description">
         <string>Paint your Terrain.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="showWorldFittingTab">
        <property name="text">
         <string>World Fitting</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/toplevel-navigation/resources/oxygen_gpl_ruler.png</normaloff>:/toplevel-navigation/resources/oxygen_gpl_ruler.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="description">
         <string>Total Extent and Detail-Level.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="terrainTaskStackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="TerrainFormation" name="formation">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="TerrainTexturing" name="texturing"/>
     <widget class="TerrainFitting" name="fitting"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TerrainFormation</class>
   <extends>QWidget</extends>
   <header>terrainformation.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TerrainTexturing</class>
   <extends>QWidget</extends>
   <header>terraintexturing.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TerrainFitting</class>
   <extends>QWidget</extends>
   <header>terrainfitting.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
